pcdsdevices/epics_motor.py

```diff
@@ -601,7 +603,13 @@ def set_current_position(self, pos):
         """
         self.set_use_switch.put(1, wait=True)
         try:
-            set_and_wait(self.user_setpoint, pos, timeout=1)
+            # Force to ignore limits
+            self.user_setpoint.put(pos, force=True)
+            # Instead of waiting on the put (broken), manually check
```

Comment:
For my own understanding, what we're doing here is putting the motor into set-mode and setting the user_setpoint to something that could be outside of the limits.  

Does this have the possibility of putting us into a weird state where the setpoint is outside the limits?

Reply:
When you change the set position using set mode, the limits also get updated appropriately to maintain the motor's apparent distance from them.

Reply:
And python doesn't pick up on this until after the set move I guess.  I see

Reply:
Yeah, the Python is actually unnecessarily limiting the setpoint update here- it sees that there are limits on the setpoint PV, but the IOC has no intention of enforcing them while in the setter mode.

Reply:
It's also worth noting also that all other version of this setter code use `force=True`, it must have snuck in as a mistake when re-implementing this here.